torch/linalg/__init__.py

```diff
@@ -148,23 +148,30 @@
 """)
 
 slogdet = _add_docstr(_linalg.linalg_slogdet, r"""
-linalg.slogdet(input) -> (Tensor, Tensor)
+linalg.slogdet(input, *, out=None) -> (Tensor, Tensor)
 
 Calculates the sign and natural logarithm of the absolute value of a square matrix's determinant,
-or of the absolute values of the determinants of a batch of square matrices :attr`input`.
+or of the absolute values of the determinants of a batch of square matrices :attr:`input`.
 The determinant can be computed with ``sign * exp(logabsdet)``.
 
 Supports input of float, double, cfloat and cdouble datatypes.
 
-.. note:: When given inputs on a CUDA device, this function synchronizes that device with the CPU.
+.. note:: When given inputs on a CUDA device, this function synchronizes that device with the CPU
```

Comment:
I wouldn't add "while checking if the call to MAGMA was successful using the info argument." because "info" is an implementation detail the reader may not be familiar with. Also the MAGMA algorithm may run on CPU.